docs/approvals-service.md

```diff
@@ -8,36 +8,59 @@ Helper functions for handling form submission approvals
 import { approvalsService } from '@oneblink/apps'
 ```
 
+- [`FormApprovalFlowStep`](#formapprovalflowstep)
+- [`FormApprovalFlowInstance`](#formapprovalflowinstance)
 - [`FormSubmissionApproval`](#formsubmissionapproval)
 - [`getFormSubmissionApprovals()`](#getformsubmissionapprovals)
 - [`getFormSubmissionApproval()`](#getformsubmissionapproval)
 - [`updateFormSubmissionApproval()`](#updateformsubmissionapproval)
 - [`retrieveFormSubmissionApprovalSubmission()`](#retrieveformsubmissionapprovalsubmission)
 
+### FormApprovalFlowStep
+
+| Property   | Type       | Description                                                              |
+| ---------- | ---------- | ------------------------------------------------------------------------ |
+| `type`     | `'SINGLE'` | The type of step                                                         |
```

Comment:
Should be type `string` and list the valid options in the description IMO. That is how we have done this elsewhere